content/reference/v1beta1/application.md

```diff
@@ -183,4 +183,6 @@ The minimum KOTS version that is required by the release.
 
 Specifying this in the application spec of the release will enforce compatibility checks for both new installations and application updates, and will block if the currently deployed KOTS version is less than the specified minimum KOTS version.
 
-This feature is not currently supported for channels that have [semantic versioning](/vendor/packaging/promoting-releases/#semantic-versioning) enabled.
+This feature is not currently supported for channels that have [semantic versioning](/vendor/packaging/promoting-releases/#semantic-versioning) enabled, nor is it supported in general if the minimum version increases and then decreases from one release to the next. In both cases this is because the intermediate releases will get lost and only the most recent release will be retrieved.
+
+When promoting a new release that specifies a minimum KOTS version that is greater than what an end user currently has deployed, that release will not appear in the version history of the admin console after checking for updates. An error message will be displayed temporarily, but it will disappear after a few minutes. This error is also displayed when checking for updates using the [`kots upstream upgrade` command](/kots-cli/upstream/). End users must update their admin console to at least the minimum version in order to fetch the update without error. KOTS cannot update itself automatically, and there is no way to update KOTS from the admin console. When increasing the specified minimum KOTS version, vendors should tell their end users to download the appropriate version of the `kots` CLI and use the [`kots admin-console upgrade` command](/kots-cli/admin-console/upgrade/) to upgrade KOTS before retrieving the updated application version.
```

Comment:
> When increasing the specified minimum KOTS version, vendors should tell their end users to download the appropriate version of the `kots` CLI and use the [`kots admin-console upgrade` command](/kots-cli/admin-console/upgrade/) to upgrade KOTS before retrieving the updated application version.

a couple of comments on this sentence:
* downloading the cli and running that command is only for existing cluster installs.
* should the vendor really tell their end users to upgrade kots? isn't that the purpose of the error message? end users should _refer_ to the vendor/vendor's docs for instructions on how to upgrade kots i feel.

Reply:
1. Good point, I'll make an adjustment to mention the existing and embedded cluster options.
2. Yes, that's the purpose of the error message, but the error message is fleeting. If someone has auto check for updates enabled, they might never see that message. Seems like the vendor should make sure their end users know about the need to update until we can have a clearer way of communicating the need to upgrade in the admin console (or updating KOTS automatically).